examples/cache-basics/cache_basics.md

```diff
@@ -1,10 +1,14 @@
 # Cache basics
 
-The Ballerina Cache API provides an in-memory cache implementation by default with a
-`Least Recently Used` algorithm-based eviction policy.
-For more information on the underlying module,
-see the [Cache module](https://lib.ballerina.io/ballerina/cache/latest/).
+The `cache` library provides an in-memory cache implementation APIs and 
+uses the `Least Recently Used` algorithm-based eviction policy.
+
+For more information on the underlying module, see the [`cache` module](https://lib.ballerina.io/ballerina/cache/latest/).
+
+This example illustrates basic operations provided by the `cache` library.
 
 ::: code cache_basics.bal :::
 
+To run this sample, use the `bal run` command.
```

Comment:
```suggestion
To run this sample, use the `bal run` command.
```
The command should be added in the `.out` file with the $sign. Please check and fix all the occurrences.

Reply:
All the commands added in the `.out` file with the $sign